### ISSUE TITLE
Login screen ui bug 

### ISSUE THREAD

**Issue by jesterro** [telegramdesktop/tdesktop#2904]
Annoying thin line between _introBg_ and _introCoverIconsFg_ 

![image_2017-01-21_16-32-05](https://cloud.githubusercontent.com/assets/25265070/22174993/11464f66-dffd-11e6-8323-cef8f1aea19c.png)

```
// intro
introBg: COLOR_DARK;
introTitleFg: windowBoldFg;
introDescriptionFg: COLOR_GRAY;
introErrorFg: COLOR_GRAY;

introCoverTopBg: #1B1B1B;
introCoverBottomBg: #FF9900;
introCoverIconsFg: #1B1B1B;
introCoverPlaneTrace: #ffffff22;
introCoverPlaneInner: #FFA31A;
introCoverPlaneOuter: #8b4b06;
introCoverPlaneTop: #f5e2c8;
```

Telegram Desktop v.1.0.2 alpha

**Comment by john-preston** [comment]
@jesterro I cannot reproduce that. What OS / interface scale do you try this in? Maybe you launched it on non retina screen and then view on retina screen or vice versa?

**Comment by jesterro** [comment]
@john-preston the screen above is from Win10 machine. Right now I'm under macOS 10.11.6 with 1280 x 800 (so no retina at all) and have the same issue.
I got feedback from community now and it's definitely not a bug of my theme.
This line appears at 100% Interface scale and disappears at 125%.

Just set `introBg` and `introCoverIconsFg`  both to black and `introCoverBottomBg` to white to see it

**Comment by auchri** [comment]
@jesterro Please upload a theme with this properties

**Comment by john-preston** [comment]
@jesterro You're absolutely right, I see the glitch in the sprite. Thanks for your report, no need to upload theme anymore.

**Comment by jesterro** [comment]
Great! Thanks.

**Comment by john-preston** [comment]
Should be fixed in the 1.0.12 release.
